modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -116,6 +116,9 @@ param availabilityZone int = 0
 @description('Required. Configures NICs and PIPs.')
 param nicConfigurations array
 
+@description('Optional. If a domain name label is specified, an A DNS record is created for the public IP in the Microsoft Azure DNS system.')
+param domainNameLabel string = ''
```

Comment:
Hey @ehanlon thank you for your contribution. There one thing: Could you please run the `utilities/tools/Set-ModuleReadMe.ps1` function for this file? It will automatically update the `readme.md` for the module. :)